### PR TITLE
Add eslint and prettier

### PR DESCRIPTION
**Pull request by emilcheva**
Configured linting and formatting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,10 @@
   "description": "back-end for the Cat Wiki challenge",
   "scripts": {
     "start": "node server",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "prettier --write './**/*.{js,json}' --config ./.prettierrc",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix"
```
Comment (steliyan) [inline package.json:10]: Does this work without specifying a working dir as in the `lint` script?

File: package.json [hunk 2]
```diff
@@ -4,7 +4,10 @@
   "description": "back-end for the Cat Wiki challenge",
   "scripts": {
     "start": "node server",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "prettier --write './**/*.{js,json}' --config ./.prettierrc",
```
Comment (steliyan) [inline package.json]: Is the `--config` arg necessary?

File: server/resolvers.js [hunk 3]
```diff
@@ -7,14 +7,15 @@ const resolvers = {
...
+    }
   },
   Breed: {
     // get all images for a breed, given a breed id, also get 9 unique images
-    breedImage : ({ id }, _, { dataSources }) => {
-      return dataSources.catAPI.getImagesOfBreed(id, limit = 9);
-    },
-  },
+    breedImage: ({ id }, _, { dataSources }) => {
+      /*eslint no-undef: "off"*/
```
Comment (steliyan) [inline server/resolvers.js]: Why?

Comment (emilcheva) [reply server/resolvers.js]: the 'limit' is not defined but still needed as an arg :/

File: package.json (hunk 2 above)
Comment (emilcheva) [reply package.json]: not really , I'm removing it ;)

File: server/resolvers.js (hunk 3 above)
Comment (steliyan) [reply server/resolvers.js]: So what's expected to happen if limit is not defined?

Comment (emilcheva) [reply server/resolvers.js]: it's optional in the API anyway, without it, we'll have all the images for the breed returned ... with limi set, we are presetting max to be returned

Comment (steliyan) [reply server/resolvers.js]: So why not just pass `undefined`/`null`/`-1` or whatever? ESLint is rightly emitting an error as such var is not defined.

**Review by steliyan — APPROVED** [review]

**Pull request merged**